clang/docs/UsersManual.rst

```diff
@@ -1681,19 +1681,27 @@ for more details.
    permitted to produce more precise results than performing the same
    operations separately.
 
-   The C standard permits intermediate floating-point results within an
+   The C/C++ standard permits intermediate floating-point results within an
```

Comment:
```suggestion
   The C/C++ standards permit intermediate floating-point results within an
```

Reply:
I'd prefer `C and C++ standards`. Same elsewhere in the changes. :-)